src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int longSide;
+    private int shortSide;
+    private int height;
+
+    public IsoscelesTrapezoid(int longSide, int shortSide, int height, String color) {
+        super(color);
+        this.longSide = longSide;
+        this.shortSide = shortSide;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) (longSide + shortSide) / 2 * height;
```

Comment:
Consider storing the result of the area calculation in a local variable before returning to improve readability. This also helps when you need to debug and inspect the area value.